zebrad/src/config.rs

```diff
@@ -105,6 +105,10 @@ pub struct TracingSection {
     /// If you provide a path with an extension the extension will be ignored and
     /// replaced with `.folded` and `.svg` for the respective files.
     pub flamegraph: Option<PathBuf>,
+
+    /// The journald flag is used for activating of sending tracing events to
+    /// systemd-journald, on Linux distributions that use systemd.
+    pub journald: bool,
```

Comment:
Style: let's try to match `use_color` here:
```suggestion
    /// Whether to send tracing events to systemd-journald, on Linux
    /// distributions that use systemd.
    pub use_journald: bool,
```

Reply:
Changed.